pkg/blob/controllerserver.go

```diff
@@ -817,7 +817,9 @@ func generateSASToken(accountName, accountKey, storageEndpointSuffix string, exp
 	sasURL, err := serviceClient.GetSASURL(
 		sas.AccountResourceTypes{Object: true, Service: false, Container: true},
 		sas.AccountPermissions{Read: true, List: true, Write: true},
-		sas.AccountServices{Blob: true}, time.Now(), time.Now().Add(time.Duration(expiryTime)*time.Minute))
```

Comment:
you are missing `sas.AccountServices{Blob: true}, time.Now()` here.
cc @umagnus

Reply:
the client is bumped to 1.5

Reply:
so `sas.AccountServices` is removed in 1.5 ? @umagnus can you check whether there would be issue?

Reply:
in v1.1 is always blob, I think it wouldn't be issue
https://github.com/Azure/azure-sdk-for-go/blob/48a06486ac095e7bd94e3527ca5c43465b87e102/sdk/storage/azblob/sas/account.go#L64C17-L64C17